tests/unit/anchore_engine/services/policy_engine/engine/feeds/test_download.py

```diff
@@ -81,77 +102,151 @@ def feed_json_file_array():
         files.append(f)
 
     return files
 
 
-def test_file_iterator(feed_json_file):
-    counter = 0
-    with FeedDataFileJsonIterator(feed_json_file.name) as jsonitr:
-        for record in jsonitr:
+class TestLocalFeedDataRepo:
+    def test_file_iterator(self, feed_json_file):
+        counter = 0
+        with FeedDataFileJsonIterator(feed_json_file.name) as jsonitr:
+            for record in jsonitr:
+                logger.debug("Record {}".format(record))
+                assert record == SIMPLE_DATA["data"][counter]
+                counter += 1
+
+        feed_json_file.close()
+        assert counter == len(SIMPLE_DATA.get("data"))
+
+    def test_fileset_iterator(self, feed_json_file_array):
+        last_rec = 0
+        itr = FileListJsonIterator([x.name for x in feed_json_file_array], 0)
+        for record in itr:
             logger.debug("Record {}".format(record))
-            assert record == simple_data["data"][counter]
-            counter += 1
-
-    feed_json_file.close()
-    assert counter == len(simple_data.get("data"))
-
-
-def test_fileset_iterator(feed_json_file_array):
-    last_rec = 0
-    itr = FileListJsonIterator([x.name for x in feed_json_file_array], 0)
-    for record in itr:
-        logger.debug("Record {}".format(record))
-        idx = int(record["vulnerability"][-1])
-        assert last_rec < idx
-        last_rec = idx
-
-    # Cleanup
-    for f in feed_json_file_array:
-        f.close()
-
-
-def test_LocalFeedDataRepo():
-    tmpdir = tempfile.mkdtemp(prefix="anchoretest_repo-")
-    r = LocalFeedDataRepo(metadata=LocalFeedDataRepoMetadata(data_write_dir=tmpdir))
-    try:
-        assert os.listdir(tmpdir) == []
-
-        r.initialize()
-        assert os.listdir(tmpdir) == ["metadata.json"]
-        ts = ts = datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc)
-        r.metadata.download_result = DownloadOperationResult(
-            started=ts, status=FeedDownloader.State.in_progress.value, results=[]
-        )
-        r.flush_metadata()
-        r.metadata = None
-        r.reload_metadata()
-        assert r.metadata.download_result.started == ts
-        assert (
-            r.metadata.download_result.status == FeedDownloader.State.in_progress.value
-        )
-
-        r.write_data(
-            "feed1",
-            "group1",
-            chunk_id=0,
-            data=b'{"next_token": "something", "data": [{"somekey": "somevalue"}]}',
-        )
-
-        with timer("Read single record group", log_level="info"):
+            idx = int(record["vulnerability"][-1])
+            assert last_rec < idx
+            last_rec = idx
+
+        # Cleanup
+        for f in feed_json_file_array:
+            f.close()
+
+    def test_local_feed_data_repo(self):
+        tmpdir = tempfile.mkdtemp(prefix="anchoretest_repo-")
+        r = LocalFeedDataRepo(metadata=LocalFeedDataRepoMetadata(data_write_dir=tmpdir))
+        try:
+            assert os.listdir(tmpdir) == []
```

Comment:
I would be surprised if this is ever anything but empty. In your testing, was this every not empty?

Reply:
not sure. this test was already written, i just refactored the module to use a class instead of standalone test methods. but i will check the tests now that you brought my attn to them

Reply:
fixed. will bring changes into PR shortly